server/ansible/roles/pbench-server-vars/tasks/main.yml

```diff
@@ -185,6 +185,22 @@
     msg: "{{ pbench_host }}"
     verbosity: 1
 
+# pbench server port
+- name: fetch the pbench server API port
+  shell: "pbench-config rest_port pbench-server"
```

Comment:
Is the port being used by the code besides this ansible role?  If not, seems like this should just be an ansible variable that is required.  If it is going to be used, then it is not clear why we need to fetch this value from the configuration file?  It would have to be set by the administrator deploying that config file anyways, right?

Reply:
It's used by the gunicorn launch.